tsfile/src/main/java/org/apache/iotdb/tsfile/utils/ReadWriteForEncodingUtils.java

```diff
@@ -98,14 +98,24 @@ public static int getLongMaxBitWidth(List<Long> list) {
   public static int readUnsignedVarInt(InputStream in) throws IOException {
```

Comment:
modify javadoc: 
@return -1 or int(>=0)

Reply:
by the way, how about ` public static int readUnsignedVarInt(ByteBuffer buffer)` (line 125)

Reply:
Even if the actual value is negative, readUnsignedVarInt() will not return -1, it will return a unknown positive num.